lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
+	// MemoryMinMiB - the minimum allowable container memory size
+	MemoryMinMiB = 64
+	// MemoryMaxMiB - the maximum allowable container memory size
+	MemoryMaxMiB = 16384
```

Comment:
Can't we calculate Max values (for cpu/mem) on runtime - based on the host that we are on?